src/roll.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+class Roll {
+
+  static ERROR_ROLL_GREATER_THAN_10 = `Roll can't be greater than 10`;
+  static ERROR_ROLL_NEGATIVE = `Roll can't be negative`;
+
+  constructor(pinsDown) {
```

Comment:
I like this use of static above, also your code is very readable and easy to follow which is great